src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -803,30 +803,38 @@ g.test('depth_stencil_attachment')
 g.test('depth_stencil_attachment,depth_clear_value')
   .desc(
     `
-  Test that depthClearValue is invalid if the value is out of the range(0.0 and 1.0) when
+  Test that depthClearValue is invalid if the value is out of the range(0.0 and 1.0) only when
   depthLoadOp is 'clear'.
   `
   )
-  .paramsSimple([
-    { depthClearValue: -1.0 },
-    { depthClearValue: 0.0 },
-    { depthClearValue: 0.5 },
-    { depthClearValue: 1.0 },
-    { depthClearValue: 1.5 },
-  ])
+  .params(u =>
+    u
+      .combine('depthLoadOp', ['load', 'clear'] as const)
+      .combineWithParams([
+        { depthClearValue: -1.0 },
+        { depthClearValue: 0.0 },
+        { depthClearValue: 0.5 },
+        { depthClearValue: 1.0 },
+        { depthClearValue: 1.5 },
+      ])
+  )
   .fn(t => {
-    const { depthClearValue } = t.params;
+    const { depthLoadOp, depthClearValue } = t.params;
 
     const depthStencilTexture = t.createTexture({ format: 'depth24plus-stencil8' });
     const depthStencilAttachment = t.getDepthStencilAttachment(depthStencilTexture);
     depthStencilAttachment.depthClearValue = depthClearValue;
+    depthStencilAttachment.depthLoadOp = depthLoadOp;
 
     const descriptor = {
       colorAttachments: [t.getColorAttachment(t.createTexture())],
       depthStencilAttachment,
     };
 
-    t.tryRenderPass(depthClearValue >= 0.0 && depthClearValue <= 1.0, descriptor);
+    const isValid =
+      depthLoadOp === 'clear' && (depthClearValue < 0.0 || depthClearValue > 1.0) ? false : true;
```

Comment:
nit: `x ? false : true` is the same as `!x`

Reply:
ok, and, it's likely we need to add `!(depthLoadOp === undefined)` because it should be invalid when depthLoadOp is `undefined`

Reply:
If it's invalid in that case then we're not successfully testing that the depthClearValue validation is skipped in this case (because we can't tell). There must be some other thing that needs to be different (other than the depth/stencil attachment format).

Reply:
Ah, I ran the test and it says:

```
EXCEPTION: Unexpected validation error occurred: depthStoreOp (StoreOp::Store) must not be set if the attachment ([TextureView]) has no depth aspect or depthReadOnly (0) is true.
 - While validating depthStencilAttachment.
 - While encoding [CommandEncoder].BeginRenderPass([RenderPassDescriptor]).
```

So if we're omitting the depth aspect we must also omit the depthStoreOp.

Reply:
Ah, thank you for finding it. Fixed.